src/sentry/static/sentry/app/components/organizations/headerItem.tsx

```diff
@@ -158,4 +173,6 @@ const StyledLock = styled(InlineSvg)`
   stroke-width: 1.5;
 `;
 
-export default React.forwardRef((props, ref) => <HeaderItem {...props} innerRef={ref} />);
+export default React.forwardRef((props: Props, ref: React.Ref<any>) => (
```

Comment:
ref gets passed to `StyledHeaderItem` - so should it be `React.Ref<HTMLDivElement>`?